### ISSUE TITLE
Add options

### ISSUE THREAD

**Issue by stormpython** [stormpython/heatmap#11]
Chart options need to be added to allow for user configuration.

List of options to add:
- [x] Ability to set margins
- [x] Padding between squares
- [x] Color options
- [x] Ability to change stroke-width and stroke-color for squares
- [ ] ~~Ability to specify opacity for squares~~
- [ ] ~~Ability to specify a scale for colors, e.g. linear, log, square root, quantile, quantize, etc.~~
- [x] Axis filtering option
- [ ] ~~Axis rotating option~~
- [ ] ~~Options to adjust width and height of visualization~~

**This may not be a complete list, but it serves as a minimal number of options to have.**

**Comment by Crazyigor1987** [comment]
Hi stormpython,
let me first thank you for the great plugin you are working on. It's the best plugin for deep analyses.

Is it an option to add horizontal and vertical scrolling feature for large results?
The visualisation is limited due the window size. It would be very helpful to add this option to prevent such compromised results:

![bug](https://cloud.githubusercontent.com/assets/16761737/14389147/8c73a8ac-fdb1-11e5-903a-13a2dbbebac2.png)

**Comment by stormpython** [comment]
@Crazyigor1987 sure, I can add the options to adjust the width and the height to allow for scrolling.

I envision that it will simply be a text box or a slider (or both) that allows you to set the width and height of the heatmap which should allow for scrolling in cases like the visualization above.

**Comment by stormpython** [comment]
1. I removed the `opacity` option because the Synthia Color Brewer scales really take care of that for us.
2. I removed the `color scale` option due to time constraints. What is in currently works and this can be added as an enhancement later on.
3. I removed the rotating axis labels option because this happens by default when labels overlap with one another. Perhaps this too can be something that is enhanced later on.
